CONTRIBUTING.md

```diff
@@ -72,14 +72,14 @@ See official pip documentation for examples and explanations: https://pip.pypa.i
 
 ```sh
 cd /path/to/evidently_repo
-pip install -e ".[dev]"
+pip install -e ".[dev,spark]"
```

Comment:
I don't think we need to have `spark` extra here. I mean, yes, we have a few extras but only `dev` is useful for like contributing

Reply:
Ok, I'll take it out. 
In my environment, the pytest fails without it.

Reply:
> Ok, I'll take it out. In my environment, the pytest fails without it.

Hm... Okay
I think we have to improve our docs about how to run tests :)
Thanks for info about it